### DISCUSSION TITLE
Google Drive Support

### DISCUSSION THREAD

**Discussion by daylightstudio** [spatie/laravel-medialibrary#2393] [Ideas]
I apologize for the double post, here and in the discussion, but I realized this may be better suited as an issue (maybe not?). In short, I encountered an issue while implementing Google Drive as a filesystem and I think there is an easy solution to fix that which is explained here:

https://github.com/spatie/laravel-medialibrary/discussions/2380

**Comment by fdelvals** [comment]
Same here. I don't have enough knowledge to fix it...

**Comment by erikn69** [comment]
I'm using with [masbug/flysystem-google-drive-ext](https://github.com/masbug/flysystem-google-drive-ext) without problems
